pkg/tests/apis/openapi_test.go

```diff
@@ -67,9 +67,6 @@ func TestIntegrationOpenAPIs(t *testing.T) {
 	dir := "openapi_snapshots"
 
 	var groups = []schema.GroupVersion{{
-		Group:   "dashboard.grafana.app",
```

Comment:
removing this because unused intentionally has differnt behavior when running in dev-mode